peekingduck/cli.py

```diff
@@ -124,13 +124,17 @@ def run(
 
     curr_dir = _get_cwd()
     if config_path is None:
-        run_config_path = curr_dir / "run_config.yml"
-    else:
-        run_config_path = Path(config_path)
+        if Path(curr_dir / "pipeline.yml").is_file():
+            config_path = curr_dir / "pipeline.yml"
+        elif Path(curr_dir / "run_config.yml").is_file():
+            config_path = curr_dir / "run_config.yml"
+        else:
+            config_path = curr_dir / "pipeline.yml"
+    pipeline_path = Path(config_path)
```

Comment:
Similarto above, will checking 
`if not (curr_dir / "pipeline.yml").is_file():` first achieve the same result with less code?

Reply:
See above.